.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+      "@typescript-eslint",
+      "prettier"
+    ],
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended",
+      "prettier"
+    ],
+    "rules": {
```

Comment:
@pirobtumen I would add to the rules, the rules of not allowing console and debugger, don't you think? By default, it would let you make commits without checking this.

Reply:
That's a good idea. Added!

Reply:
👍🏻 🙃